packages/docusaurus/src/client/theme-fallback/Error/index.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 // Should we translate theme-fallback?
-/* eslint-disable @docusaurus/no-untranslated-text */
+/* eslint-disable @docusaurus/no-untranslated-text, @docusaurus/prefer-docusaurus-heading */
```

Comment:
similarly I think core should not use this rule: add global config override instead of comments

Reply:
Sure, it makes sense. Should I add an override to disable this rule in `.eslintrc.js`?

Reply:
yes 👍